aclocal.m4

```diff
@@ -1,6 +1,6 @@
-# generated automatically by aclocal 1.16.1 -*- Autoconf -*-
+# generated automatically by aclocal 1.15 -*- Autoconf -*-
```

Comment:
Might want to remove these aclocal.m4 changes since they seem to be irrelevant.

Reply:
these are autotools-generated, then just not need to be reviewed.
(I guess the only reason cpython keeps these in the repo is convenience for people working
on autotools-less systems, for one way or another).

Reply:
ok, all tidy now, kept the old aclocal.